Linear_Regression/LinearRegression Exercises.ipynb

```diff
@@ -0,0 +1,1442 @@
+{
```

Comment:
<ul><li>Same copy pasting artefact as previously</li><li><code>as adelie_regression</code> --&gt; not anymore ;) </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>